src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall() {
+        int random = new Random().nextInt(100);
```

Comment:
lets make Random a class-level variable - to avoid creating a new instance of Random each time this method is invoked

avoid magic numbers - use constants